packages/ibm-products/src/components/SidePanel/SidePanel.tsx

```diff
@@ -728,6 +734,9 @@ export let SidePanel = React.forwardRef(
               className={`${blockClass}__close-button`}
               label={closeIconDescription}
               onClick={onRequestClose}
+              onKeyDown={(event) =>
+                slideIn ? undefined : handleEscapeKey(event)
+              }
               title={closeIconDescription}
```

Comment:
can  we simply this?
`onKeyDown={slideIn ? undefined : handleEscapeKey}`

Reply:
yees thanks! just simplified